pkg/util/objectwatcher/objectwatcher.go

```diff
@@ -80,6 +87,31 @@ func (o *objectWatcherImpl) Create(cluster *v1alpha1.Cluster, desireObj *unstruc
 	return nil
 }
 
+// resolveAlreadyExist used to resolve the 'AlreadyExist' conflicts in a 'Create' process.
+// The conflict may happen in following known scenarios:
+// - 1. In a reconcile process, the execution controller successfully applied resource to member cluster but failed to update the work conditions(Applied=True),
+//   when reconcile again, the controller will try to apply(by create) the resource again.
+// - 2. The resource already exist in the member cluster but it's not created by karmada.
+func (o *objectWatcherImpl) resolveAlreadyExist(dynamicClusterClient *util.DynamicClusterClient, gvr schema.GroupVersionResource, desireObj *unstructured.Unstructured) error {
+	existObj, err := dynamicClusterClient.DynamicClientSet.Resource(gvr).Namespace(desireObj.GetNamespace()).Get(context.TODO(), desireObj.GetName(), metav1.GetOptions{})
+	if err != nil {
+		return fmt.Errorf("failed to get exist resource %s(%s/%s): %v", desireObj.GetKind(), desireObj.GetNamespace(), desireObj.GetName(), err)
+	}
+
+	// Avoid updating resources that not managed by karmada.
+	if util.GetLabelValue(desireObj.GetLabels(), util.WorkNameLabel) != util.GetLabelValue(existObj.GetLabels(), util.WorkNameLabel) {
+		return fmt.Errorf("resource %s(%s/%s) already exist but not managed by karamda", desireObj.GetKind(), desireObj.GetNamespace(), desireObj.GetName())
+	}
```

Comment:
In this case, we need to notify the controller without retrying and annotating the failed reason somewhere, like condition.

Reply:
Can you help to open an issue to track this?